style.css

```diff
@@ -65,18 +84,18 @@ button:enabled:active {
 /* Headline Section */
 
 .headline {
-  list-style: none;
   min-height: 590px;
   height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   background-color: #1c1a19;
-  background-image: url('assets/images/illustration-header-mobile.png');
+  background-image: url('assets/images/illustration-header.png');
   background-repeat: no-repeat;
-  background-position-x: calc(100% + 325px);
-  background-position-y: 118%;
+  background-position-x: calc(100% + 375px);
+  background-position-y: 35vh;
+  background-size: 85vh;
```

Comment:
- [ ] Kindly fix the background image issue for the `heading.` I can't see any `background image` in the desktop version. It is a core requirement to follow the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=86lhUD7Kl0qMQAIy-0) as close as possible.
![Screenshot from 2023-01-20 00-03-07](https://user-images.githubusercontent.com/60083437/213525090-71d5e3c8-c7a6-42fc-95c8-80d1e1455c39.png)

Reply:
That is really odd. @Tufahel can you tell me the dimensions of your screen. I'm able to see the image in my laptop screen without any issues.

Reply:
@jlvFlores, I am not sure what the issue is. But I can't see the background image. You have used it, and while inspecting, I can see that you have used it. [Video](https://www.loom.com/share/c7975fa1203b46df8b4dd558f2fa446d)

![Screenshot from 2023-01-20 00-32-02](https://user-images.githubusercontent.com/60083437/213530397-f86109e7-b8aa-4a15-a2ba-40966abc191a.png)

Reply:
@Tufahel Do you think this is a browser issue, or maybe you didn't pull the image assets? I'm not accusing, I'm just really stump as to why it doesn't show the background image on your screen despite working on mine and presumably on the previous CR's screen too.

Reply:
Thank you so much, I'll fix it, and I'll be sure to not let it happen again.